src/analysis/processing/qgsalgorithmfilterpoints.cpp

```diff
@@ -0,0 +1,194 @@
+/***************************************************************************
+                         qgsalgorithmfilterpoints.cpp
```

Comment:
I'd rename the relevant classes to qgsalgoriithmfiltervertex (or qgsalgorithmfiltervertices), to avoid any possible confusion.

Reply:
Done!